Makefile

```diff
@@ -36,6 +36,12 @@ distclean:
 	$(RM) -r $(BUILDDIR) $(TARGETS)
 	$(ECHO) " done"
 
+BUSTED_TESTS:=$(wildcard spec/**/*.lua)
```

Comment:
Doesn't work here:
```
Makefile:11: *** spec/menubar/icon_theme_spec.lua spec/awful/screen_spec.lua spec/awful/keyboardlayout_spec.lua spec/gears/geometry_spec.lua spec/gears/protected_call_spec.lua spec/gears/string_spec.lua spec/gears/table_spec.lua spec/gears/object_spec.lua spec/gears/cache_spec.lua spec/gears/color_spec.lua spec/gears/matrix_spec.lua spec/wibox/test_utils.lua spec/wibox/hierarchy_spec.lua.  Schluss.
```
It just does `$(wildcard spec/*/*.lua)`, the second `*` doesn't change anything. For example, `spec/preload.lua` is missing. This is not recursive.